core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -88,7 +88,8 @@ private[ui] class AllJobsPage(parent: JobsTab, store: AppStatusStore) extends We
       // The timeline library treats contents as HTML, so we have to escape them. We need to add
       // extra layers of escaping in order to embed this in a Javascript string literal.
       val escapedDesc = Utility.escape(jobDescription)
-      val jsEscapedDesc = StringEscapeUtils.escapeEcmaScript(escapedDesc)
+      val jsEscapedDescForTooltip = StringEscapeUtils.escapeEcmaScript(Utility.escape(escapedDesc))
```

Comment:
Ur, this is triple escaping since we are using `escapedDesc`?
Technically, this is equivalent `StringEscapeUtils.escapeEcmaScript(Utility.escape(Utility.escape(jobDescription)))`.

Reply:
If we'd like to get a tooltip text `<console>`, the value of the  corresponding attribute `data-title` should be `&lt;console&gt;`.
With single `Utility.escape`, the value of attribute will be `<console>` so additional `Utility.escape` is needed.

Reply:
Got it. Thanks~